app/src/main/java/com/andrew/tasky/agenda/presentation/notifications/AgendaNotificationService.kt

```diff
@@ -0,0 +1,49 @@
+package com.andrew.tasky.agenda.presentation.notifications
+
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import androidx.core.app.NotificationCompat
+import androidx.navigation.NavDeepLinkBuilder
+import com.andrew.tasky.R
+import com.andrew.tasky.core.MainActivity
+
+class AgendaNotificationService(
+    private val context: Context
+) {
+
+    private val notificationManager = context
+        .getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+
+    fun showEventNotification(title: String, description: String) {
```

Comment:
Having multiple of such functions for all types of agenda items will be repetitive. How about passing in what differs in form of parameters? For example:
```
fun showAgendaNotification(info: AgendaNotificationInfo) {
    // ...
}

data class AgendaNotificationInfo(
    val title: String,
    val description: String,
    val notificationChannel: String,
    val navDestinationId: Int,
)
```